.gitignore

```diff
@@ -5,4 +5,6 @@ debugger
 cli
 *.sw*
 mochawesome-report
-*lock*
\ No newline at end of file
+*lock*
+*.DS_Store
+package-lock.json
```

Comment:
Yeah probably a good idea.

Reply:
While in my personal experience, ignoring the `package-lock.json` file has never caused a problem, ignoring it in git is not recommended in the official documentation. I would consider the following before merging:

[npm's documentation on package-lock.json](https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json) states that one of the main reasons to commit it is that it's used to
> Describe a single representation of a dependency tree such that teammates, deployments, and continuous integration are guaranteed to install exactly the same dependencies.

[Node's documentation](https://nodejs.dev/learn/the-package-lock-json-file) states that "The package-lock.json file needs to be committed to your Git repository"

I think it is fine to add it to the git ignore, but we have to remember that if/when people `npm install` after any changes to `package.json`, they could install different versions of dependencies than everyone else. There is a very slim possibility this could cause issues.

Reply:
Sounds reasonable. I added `package-lock.json` back to git.

Reply:
Agree – it's good to have in .gitignore (because it stops accidental huge diffs), but I'd like it to be present in git. I was just going to regenerate it once after merging to get a good copy, but adding it back is great, too, thanks!

Reply:
We were different versions of `npm` and they generates different `package-lock.json`. We'll ignore it until a good copy of it is generated.

Reply:
>  it's good to have in .gitignore (because it stops accidental huge diffs), but I'd like it to be present in git. I was just going to regenerate it once after merging to get a good copy

Ah, I see. I did not consider this. I think you are right that keeping it in the .gitignore and regenerating it after merging is a good way of keeping `package-lock.json` up to date for everybody. 

Sorry for creating more work for you @rlisahuang and @madun1999